src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of
+ * related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const settings = sourceString
+    .split(';')
+    .filter(style => style.includes(':'))
+    .map(style => style.split(':'))
+    .reduce((prev, style) => ({
+      ...prev,
+      [style[0].trim()]: style[1].trim(),
```

Comment:
You can use array destructuring, it's optional but looks better
```suggestion
    .reduce((prev, [key, value]) => ({
      ...prev,
      [key.trim()]: value.trim(),
```

Reply:
thanks, fixed